Rakefile

```diff
@@ -71,6 +77,91 @@ def re_run_basic_smoke_test
   sh beaker
 end
 
+def jenkins_passing_json_parsed
```

Comment:
on second thought..  it's gonna take some refactoring to combine these helpers.  
the helpers in the acceptance lib don't have any tests, so i don't really want to mess with them in this timeframe.  

can we see if this works as is, and ticket the refactor of this stuff for later?